assets/css/main.css

```diff
@@ -446,4 +446,26 @@ body:has(#menu-controller:checked) {
   text-decoration-line: underline;
   text-decoration-thickness: 3px;
   text-underline-offset: 4px;
-}
\ No newline at end of file
+}
+
+.grid-w10 { width: calc(10% - 5px); margin: 0px !important; }
+.grid-w15 { width: calc(15% - 5px); margin: 0px !important; }
+.grid-w20 { width: calc(20% - 5px); margin: 0px !important; }
+.grid-w25 { width: calc(25% - 5px); margin: 0px !important; }
+.grid-w30 { width: calc(30% - 5px); margin: 0px !important; }
+.grid-w33 { width: calc(33% - 5px); margin: 0px !important; }
+.grid-w35 { width: calc(35% - 5px); margin: 0px !important; }
+.grid-w40 { width: calc(40% - 5px); margin: 0px !important; }
+.grid-w45 { width: calc(45% - 5px); margin: 0px !important; }
+.grid-w50 { width: calc(50% - 5px); margin: 0px !important; }
+.grid-w55 { width: calc(55% - 5px); margin: 0px !important; }
+.grid-w60 { width: calc(60% - 5px); margin: 0px !important; }
+.grid-w66 { width: calc(66% - 5px); margin: 0px !important; }
+.grid-w65 { width: calc(65% - 5px); margin: 0px !important; }
```

Comment:
Just a pet peeve of mine and maybe I have a bit of OCD, but these 2 are not in the right order 😆

Reply:
For crying ou loud... 🤣

Reply:
Ah, I tried to fix it myself but apparently I do not have permission to push to this branch.

Reply:
you can't commit directly to it, but you can fork and create a PR to the branch... but I'll fix it no issues